examples/instruct_pix2pix/train_instruct_pix2pix_sdxl.py

```diff
@@ -1109,7 +1109,7 @@ def collate_fn(examples):
             progress_bar.set_postfix(**logs)
 
             ### BEGIN: Perform validation every `validation_epochs` steps
-            if global_step % args.validation_steps == 0 or global_step == 1:
```

Comment:
Do we want to do validation at every step?

Reply:
Seems reasonable. Thank you!